controllers/profiles/reconciler_dev.go

```diff
@@ -324,11 +358,108 @@ func mountDevConfigMapsMutateVisitor(flowDefCM, propsCM *v1.ConfigMap) mutateVis
 			deployment.Spec.Template.Spec.Containers[0].VolumeMounts = make([]v1.VolumeMount, 0)
 			deployment.Spec.Template.Spec.Containers[0].VolumeMounts = volumeMounts
 
+			for keycm, cm := range externalResourceConfigMaps {
+				deployment.Spec.Template.Spec.Volumes = append(deployment.Spec.Template.Spec.Volumes,
+					v1.Volume{
+						Name: cm.Name,
+						VolumeSource: v1.VolumeSource{
+							ConfigMap: &v1.ConfigMapVolumeSource{
+								LocalObjectReference: v1.LocalObjectReference{Name: cm.Name},
+							},
+						},
+					})
+				deployment.Spec.Template.Spec.Containers[0].VolumeMounts = append(deployment.Spec.Template.Spec.Containers[0].VolumeMounts,
+					v1.VolumeMount{
+						Name:      cm.Name,
+						ReadOnly:  true,
+						MountPath: getMountPath(keycm),
+					})
+			}
 			return nil
 		}
 	}
 }
 
+func getMountPath(name string) string {
+	var mountPath = quarkusDevConfigMountPath
+	if strings.HasSuffix(name, extResCamelSuffix) {
```

Comment:
Instead, create a `map[string]string` where the Key is the expected label suffix.

Reply:
ok